package.json

```diff
@@ -7,6 +7,9 @@
   ],
   "private": true,
   "devDependencies": {
-    "lerna": "^3.2.1"
-  }
+    "lerna": "^3.19.0",
+    "tslint": "^5.20.0",
+    "typescript": "^3.7.3"
+  },
+  "name": "fints-lerna"
```

Comment:
I am unsure whether this package really needs a name. It's not intended to be published.

Reply:
I believe the package doesn't need a name, please remove it.